mlops_ml_models/deploy_model_endpoint.py

```diff
@@ -2,7 +2,7 @@
 
 
 def deploy_model(
-    model_name: str, instance_type: str, endpoint_name,
+    model_name: str, model_type: str, instance_type: str, endpoint_name,
```

Comment:
Passing in the `model_type` here allows us to pass the argument to the model container at runtime, rather than having to hardcode it into the pycaret image at build time.